test/jdk/java/net/InetAddress/getOriginalHostName.java

```diff
@@ -41,10 +41,10 @@ public class getOriginalHostName {
     public static void main(String[] args) throws Exception {
         final String HOST = "dummyserver.java.net";
         InetAddress ia = null;
-        ia = InetAddress.getByName(HOST);
-        testInetAddress(ia, HOST);
-        ia = InetAddress.getByName("255.255.255.0");
-        testInetAddress(ia, null);
+        ia = getInetAddress(HOST);
+        if (ia != null) testInetAddress(ia, HOST);
+        ia = getInetAddress("255.255.255.0");
```

Comment:
Please revert the changes on this line. UnknownHostException should not be thrown here.

Reply:
Thanks, the change of "255.255.255.0" has been reverted.